### PR TITLE
switches to note tab automatically

### PR DESCRIPTION
**Pull request by joelhooks**

### CODE REVIEW

File: src/components/EggheadPlayer/use-egghead-player.ts [hunk 1]
```diff
@@ -228,9 +252,8 @@ const defaultPlayerPreferences = {
   muted: false,
   theater: false,
   defaultView: 'transcript',
-  sideBar: {
-    activeTab: 0,
-  },
+  activeSidebarTab: 0,
```
Comment (joelhooks) [inline src/components/EggheadPlayer/use-egghead-player.ts:255]: flattened this for ease of access

File: src/pages/video-test.tsx [hunk 2]
```diff
@@ -40,9 +40,10 @@ const EggheadPlayer: React.FC<{
   notesUrl: string
 }> = ({videoResource, notesUrl}) => {
   const playerContainer = React.useRef<any>()
-  const playerPrefs = useEggheadPlayerPrefs()
+  const {setPlayerPrefs, getPlayerPrefs} = useEggheadPlayerPrefs()
   const {player} = usePlayer()
   const {activeMetadataTracks = []} = player
+  const {activeSidebarTab} = getPlayerPrefs()
```
Comment (joelhooks) [inline src/pages/video-test.tsx:46]: was having a difficult time getting this to update from the prefs hook until I made a getter which now reads at ender and updates

I thought calling `set` on a `useState` would trigger the update but for some reason we aren't getting a refreshed value even though the underlying state is update.

File: src/components/player/cue-bar.tsx [hunk 3]
```diff
@@ -97,8 +98,14 @@ const NoteCue: React.FC<any> = ({
   }
 
   React.useEffect(() => {
-    setVisible(player.activeMetadataTrackCues.includes(cue) && !player.seeking)
-  }, [player.activeMetadataTrackCues, player.seeking, cue])
+    if (!muteNotes) {
```
Comment (joelhooks) [inline src/components/player/cue-bar.tsx:101]: this adds a preference but no UI to control it, set the preference to mute!

**Review by DrShpongle — APPROVED** [review]
Cool 👍

**Pull request merged**